Makefile

```
@@ -26,8 +32,11 @@ detect_changed_source_translations:
 dummy_translations:
 	cd edx_credentials_themes && i18n_tool dummy
 
-extract_translations:
-	cd edx_credentials_themes && django-admin.py makemessages -l en -d django
+extract_translations: ## extract strings to be translated, outputting .po files
+	cd $(WORKING_DIR) && i18n_tool extract
+	mv $(EXTRACTED_DJANGO) $(EXTRACTED_TEXT)
+	sed -i'' -e 's/nplurals=INTEGER/nplurals=2/' $(EXTRACTED_TEXT)
+	sed -i'' -e 's/plural=EXPRESSION/plural=\(n != 1\)/' $(EXTRACTED_TEXT)
```

I'm not sure why this is being changed. As far as I can tell, we don't need to change this step.

The pattern above is generally used for XBlocks. Anyway, expect to clean up the output of `i18n_tools extract` since I think it needs further processing:

 - Example: https://github.com/appsembler/edx-theme-codebase/blob/juniper/main/scripts/i18n-push.sh#L39-L43

```suggestion
extract_translations:
	cd edx_credentials_themes && django-admin.py makemessages -l en -d django
```

---

donem see [my comment](https://github.com/openedx/credentials-themes/pull/613#issuecomment-1523394362) please